infra/modules/database/main.tf

```diff
@@ -432,3 +451,21 @@ resource "aws_vpc_security_group_ingress_rule" "vpc_endpoints_ingress_from_role_
   ip_protocol                  = "tcp"
   referenced_security_group_id = aws_security_group.role_manager.id
 }
+
+resource "aws_iam_role" "service" {
+  name               = "${var.name}-migrator"
+  assume_role_policy = data.aws_iam_policy_document.ecs_tasks_assume_role_policy.json
+}
```

Comment:
as specified in [the ticket](https://github.com/navapbc/template-infra/issues/354) we should be creating the role in the service layer, not the database layer

> We will also want to do something similar to pass in the migrator_access_policy_arn into the service layer so that it the module can create a migrator role the same way it [created a role for the service](https://github.com/navapbc/template-infra/blob/main/infra/modules/service/main.tf#L250-L253).